wikia/common/mw_database/query_builder.py

```diff
@@ -51,7 +51,7 @@ def insert(self, table, data, ignore_errors=False):
         if ignore_errors:
             ignore = 'IGNORE '
 
-        sql = 'INSERT {}INTO {}({}) VALUES ({});'.format(ignore, table, ', '.join(columns), ', '.join(values))
+        sql = 'INSERT {}INTO {}(`{}`) VALUES ({});'.format(ignore, table, '`, `'.join(columns), ', '.join(values))
```

Comment:
I'd rather 
```
map(lambda s: '`{}`'.format(s), columns)
``` 
so that the backticks stay in one place and the code is more readable overall.